src/Tooltip.tsx

```diff
@@ -14,7 +14,7 @@ const TooltipBase = styled.span<SystemCommonProps & SxProp>`
     display: none;
     width: 0px;
     height: 0px;
-    color: ${get('colors.black')};
+    color: ${get('colors.tooltip.bg')};
```

Comment:
Had to double check this because it's a bit weird to assign a bg to a text-color, but it checks out in primer-css:

![image](https://user-images.githubusercontent.com/16739070/110388241-649d2700-8028-11eb-9d74-b61c18b592cc.png)